scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -321,4 +320,72 @@ task TerminateWorkflow {
     preemptible: 3
     cpu: 1
   }
+
+  output {
+    Boolean done = true
+  }
+}
+
+task ScaleXYBedValues {
+    input {
+        Boolean go = true
+        File interval_weights_bed
+        Float x_bed_weight_scaling
+        Float y_bed_weight_scaling
+    }
+    command <<<
+        python3 <<FIN
```

Comment:
I should possibly make this a separate script with tests and add it to the Docker image.